docs/data/sql_functions.yml

```diff
@@ -571,6 +571,22 @@ collection:
     table: MAP.at(ANY)
     description: Returns the value specified by key value in map.
 
+json:
+  - sql: JSON_EXISTS(jsonValue, path [ { TRUE | FALSE | UNKNOWN | ERROR } ON ERROR ])
+    table: STRING.jsonExists(STRING path [, JsonExistsOnError onError])
+    description: |
+      Determines whether a JSON string satisfies a given path search criterion.
+
+      This follows the ISO/IEC TR 19075-6 specification for JSON support in SQL.
+
+      ```
+      SELECT JSON_EXISTS(
```

Comment:
can we add a couple more examples? at least one with strict mode. We should try to summarize `ISO/IEC TR 19075-6 specification` a bit. E.g. what is the default error behavior?

Reply:
One question that arises for me seeing this the first time? Does return true only if there is a value like string or is `$.items` also fine in your example?

Reply:
I've added more examples which hopefully are more clear.